SVN doesn't work with Remote-SSH extension

- VSCode Version: 1.34.0-insider
- OS Version: Windows 10 / RHEL 7
- Extension Version: 1.48.1
- System Language: English (US)
- SVN Version: 1.9.7

## Issue ##

I am using the new Remote-SSH extension. When I try to use the SVN extension with it, it reports that "There is no data provider registered that can provide view data".

I thought that it may be that I have svn installed in /opt/subversion/bin/svn, but I tried setting svn.path to that path, and it still doesn't work.

I also tried setting it to my TortoiseSVN path, in case it needs to use the executable on the Windows side for some reason. That doesn't work either.

In the developer's console, there is an error "Cannot find module 'original-fs'". I don't know if that is related.

Note that Git works fine.

## Steps to Reproduce ##

1. Install the Remote-SSH extension.
2. Connect to a Linux system
3. Attempt to use the SVN extension after opening a path that is using SVN.

## SVN Output ##
None - no option in the dropdown.

## Screenshots ##

![image](https://user-images.githubusercontent.com/2024233/57186994-0e03b680-6eae-11e9-8b75-b26b3f0c0641.png)

--- a/package.json
+++ b/package.json
@@ -48,6 +48,7 @@
   "dependencies": {
     "is-utf8": "^0.2.1",
     "minimatch": "^3.0.4",
+    "original-fs": "^1.0.0",
     "semver": "^5.7.0",
     "tmp": "0.1.0",
     "xml2js": "^0.4.19"

